core/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ add_definitions("-DSOURCE_PATH_SIZE=${SOURCE_PATH_SIZE}")
 
 if ($ENV{TARGET} MATCHES "nCipher")
   include(codesafe.cmake)
+
+  if ($ENV{UNSIGNED} MATCHES "yes")
```

Comment:
Instead of having 3 parameters, should we have a single parameter (which picks among 4 profiles)? The 4 profiles would be:

- dev-without-hsm = no HSM, btc-testnet
- dev-with-hsm = HSM, unsigned, btc-testnet
- staging = HSM, signed, btc-testnet
- prod = HSM, signed, btc-mainnet

Reply:
Yes, having profiles will be cleaner (and we can document the list above)